contrib/pg_tde/documentation/docs/release-notes/rc.md

```diff
@@ -0,0 +1,86 @@
+
+
+
+## Changelog
+
+### New Features
+ 
+* [PG-1002](https://perconadev.atlassian.net/browse/PG-1002) - Added the requirement of an explicit configuration of a WAL principal key to ensure correct WAL encryption
+
+* [PG-1234](https://perconadev.atlassian.net/browse/PG-1234) - Added functions for separate global and database key management permissions 
+
+* [PG-1255](https://perconadev.atlassian.net/browse/PG-1255) - Added functionality to delete key providers
+
+* [PG-1256](https://perconadev.atlassian.net/browse/PG-1256) - Added single-tenant support via the default principal key functionality
+
+* [PG-1258](https://perconadev.atlassian.net/browse/PG-1258) - Added functions to display additional information  about principal keys / providers 
+
+* [PG-1294](https://perconadev.atlassian.net/browse/PG-1294) - Improve WAL encryption by encrypting entire WAL files
+
+* [PG-1303](https://perconadev.atlassian.net/browse/PG-1303) - Deprecated tde_heap_basic access method
+
+## Improvements
+
+* [PG-1361](https://perconadev.atlassian.net/browse/PG-1361) - Investigated and fixed pg_tde relocatability
+
+* [PG-1367](https://perconadev.atlassian.net/browse/PG-1367) - Created a separate generic script that is used to configure server with pg_tde from the existing make-test-tde.sh script
+
+### Bugs Fixed
+
+
+* [PG-821](https://perconadev.atlassian.net/browse/PG-821) - Fixed the issue with `pg_basebackup` failing when configuring replication
+
+* [PG-847](https://perconadev.atlassian.net/browse/PG-847) - Fixed the issue with `pg_basebackup` and `pg_checksum` throwing an error on files created by `pg_tde` when the checksum is enabled on the database cluster
+
+* [PG-1004](https://perconadev.atlassian.net/browse/PG-1004) Fixed the issue with `pg_checksums` utility failing during checksum verification on `pg_tde` tables. Now `pg_checksum` skips encrypted relations by looking if the relation has a custom storage manager (SMGR) key.
+
+* [PG-1373](https://perconadev.atlassian.net/browse/PG-1373) Fixed the issue with potential unterminated strings by using the `memset()` or `palloc0()` instead of the `strncpy()` function.   
+
+* [PG-1376](https://perconadev.atlassian.net/browse/PG-1376) - Fixed the issue with getting a warning message when setting a default principal key by using the correct lock type during the function execution.
+
+* [PG-1378](https://perconadev.atlassian.net/browse/PG-1378) - Fixed the issue with toast tables created by ALTER TABLE command not being encrypted by fixing the sequence and alter table handling by event trigger 
+
+* [PG-1380](https://perconadev.atlassian.net/browse/PG-1380) Added support for `pg_tde_is_encrypted()` function on indexes and sequences
```

Comment:
This is kinda both a bug and a feature because instrad of displaying an error when used on an index it just said "false" even if it was encrypted. I think I would move this to features.